core/controllers/admin.py

```diff
@@ -672,11 +672,12 @@ def _load_dummy_new_structures_data(self) -> None:
                 'fragm')
 
             topic_1.update_meta_tag_content('dummy-meta')
-            svg_file = open(
-                'core/tests/data/thumbnail.svg', 'rt', encoding='utf8')
-            svg_file_content = svg_file.read()
-            svg_file.close()
-            raw_image = svg_file_content.encode('ascii')
+            raw_image = b''
+            with open(
+                'core/tests/data/thumbnail.svg', 'rt',
+                encoding='utf8') as svg_file:
```

Comment:
Isn't the encoding supposed to be "utf-8" rather than "utf8"?

Reply:
@seanlip it is done and also I have already added the screen recording in the PR description for backend testing part. There is a video link in PR overview section.
Here is the [link](https://github.com/oppia/oppia/assets/92575005/00a03810-700d-4f47-8627-97581854ded4) to the screen recording.